setup.py

```diff
@@ -54,7 +54,7 @@ def get_static_files(path):
 
 setup(
     name='luigi',
-    version='2.7.5.affirm.1.4.0',
+    version='2.7.5+affirm.1.4.2',
```

Comment:
.2 instead of .1?

Reply:
yeah I messed up and accidentally created this pypi package already :(